packages/quantification/src/quantification.ts

```diff
@@ -548,6 +471,36 @@ const getGrandfatheredTonneQuantities = ({
   };
 };
 
+/**
+ * Takes the year-over-year difference in soil carbon content across multiple polygons,
+ * sums the future and baseline values individually, then provides the average year-over-year
+ * difference over the first 10 years.
+ *
+ * @returns The average year-over-year change for the first 10 years after the switch year
+ */
+const getTenYearProjectedTonnesPerYear = ({
+  somscAnnualDifferencesBetweenFutureAndBaselineScenariosPerPolygon,
```

Comment:
idk if there's any way we can shorten names like this, but these are pretty hard to wrap your head around, in my opinion

Reply:
I agree, but it's difficult to change for now. We could follow up with PRs to change it but I think we'd need to get buy in from others. Any ideas on a better name?